README.md

```diff
@@ -45,6 +45,9 @@ apt install -y python3-dev libpython3-dev build-essential ocl-icd-libopencl1 \
     llvm-${LLVM_VERSION}-dev
 ```
 
+For LLVM 18 and above also `libpolly-${LLVM_VERSION}-dev libzstd-dev` are
```

Comment:
This is actually needed also for earlier LLVM(s) when using static linking, i've just checked and LLVM 17 definitely requires libpolly-17-dev; earlier LLVMs possibly but i haven't checked.

Reply:
i'll fix it manually